DataManagementSystem/Client/S3GWClient.py

```diff
@@ -0,0 +1,20 @@
+""" Client to interact with the S3GW  """
+
+from DIRAC.Core.Base.Client import Client, createClient
+
+
+@createClient('DataManagement/S3GW')
+class S3GWClient(Client):
+  """ Client code to the S3GW
+  """
+
+  def __init__(self, url=None, **kwargs):
+    """ Constructor function.
+    """
+    Client.__init__(self, **kwargs)
+    self.setServer('DataManagement/S3GW')
+    if url:
+      self.setServer(url)
+
+  def createPresignedUrl(self, storageName, methodName, objectName, expiration=3600, **kwargs):
```

Comment:
Hum, in principle, these URLs are used straight away. I can't think of a case where one would use it much later. Except maybe when generating the pool xml catalog, but that usecase is not supported yet anyway, and it would be done in a different part of the code

Reply:
Is there a way this ends up in the output of `dirac-dms-get-accessURL` or `dirac-dms-get-file`? If not, then I agree with you.

Reply:
that's the not yet supported usecase, coming next :-)

Reply:
Implemented, with 24h token lifetime :-)